exercises/concept/card-tricks/.docs/introduction.md

```diff
@@ -3,11 +3,28 @@
 Slices in Go are similar to lists or arrays in other languages. They hold a number of elements of a specific type (or interface).
 
 Slices in Go are based on arrays. Arrays have a fixed size. A slice, on the other hand, is a dynamically-sized, flexible view into the elements of an array.
-Technically a slice consists of a pointer to a start element in an array and a length. This makes for example creating a sub-slice `s[3:7]` very efficient.
-But don't worry: most of the time this knowledge is not needed in order to work with slices.
 
 A slice is written like `[]T` with `T` being the type of the elements in the slice:
 
 ```go
-var s []int
+var empty []int                 // an empty slice
+withData := []int{0,1,2,3,4,5}  // a slice pre-filled with some data
 ```
+
+You can get/set an element at a given zero-based index using square-bracket notation:
+
+```go
+withData[1] = 5
+x := withData[1] // x is now 5
+```
+
+You can create a new slice from an existing slice by getting a range of elements, once again using square-bracket notation, but specifying both a starting (inclusive) and ending (exclusive) index.
+If you don't specify a starting index, it defaults to 0.
+If you don't specify an ending index, it defaults to the length of the slice.
+
+```go
+newSlice := withData[2:4] // newSlice == []int{2,3}
+```
+
+Slices are the first reference type you are likely to encounter in Go.
```

Comment:
We will have a separate exercise teaching about pointers and with types are reference types etc. So if this information is not absolutely essential to solve the exercise it should be removed from both introduction files. It's fine to mention it in the about.md of the concept if its not already in there.